### ISSUE TITLE
Fastlane Automated Tests

### ISSUE THREAD

**Issue by FDuhen** [wzieba/Firebase-Distribution-Github-Action#131]
Since today I've got this issue when running my workflow to upload my iOS App

```
i  distributing to testers/groups...
✔  distributed to testers/groups successfully
i  starting automated tests (note: this feature is in beta)
✔  Release test created successfully
i  the automated tests results are pending

Error: An unexpected error has occurred.
```

I guess that's related to this file 
https://github.com/firebase/fastlane-plugin-firebase_app_distribution/blob/master/lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb#L396

On my latest execution (yesterday), I didn't have this log and everything was working fine.
Is it possible to hardcode the Firebase version used by this Action ?

**Comment by MaesterChestnut** [comment]
Hey there! Sorry your workflow was interrupted. We just released a fix to our API that should prevent this from erroring for you. It just released so please try your workflow again.
Also, https://github.com/firebase/firebase-tools/pull/6750 fixes the firebase command; [version 13.2.1](https://github.com/firebase/firebase-tools/releases/tag/v13.2.1) with this fix has been released.

PS: https://github.com/firebase/firebase-tools/issues is the appropriate place to report issues (like https://github.com/firebase/firebase-tools/issues/6749) with the firebase command.

**Comment by FDuhen** [comment]
Issue fixed 
https://github.com/firebase/fastlane-plugin-firebase_app_distribution/issues/365
